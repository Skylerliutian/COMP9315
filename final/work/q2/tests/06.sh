./bnl 10 2 10
