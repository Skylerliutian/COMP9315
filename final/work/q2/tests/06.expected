Running: ./bnl 10 2 10

Frames:     [00] [01] [02] [03] [04] [05] [06] [07] [08] [09]
Contents:    R08  S00  S01  R09  R02  R03  R04  R05  R06  R07
PinCount:      0    0    0    0    0    0    0    0    0    0
Popularity:    2    3    3    2    0    0    0    0    0    0
Clock: 4

#requests: 30
#releases: 30
#hits    : 18
#reads   : 12
